src/runtime/vm/executable.cc

```diff
@@ -74,6 +76,12 @@ PackedFunc Executable::GetFunction(const std::string& name, const ObjectPtr<Obje
       int index = args[1];
       *rv = this->GetFunctionParameterName(func_name, index);
     });
+  } else if (name == "vm_load_executable") {
```

Comment:
It is a factory function in C++, I am using the Executable to act directly as a factory module instead of needing to introduce a 3rd concept like the GraphRuntimeFactory does.